### PR TITLE
Fix/cypress testing

### PR DESCRIPTION
**Pull request by mietcls**
Fixes testing for Cypress and #1386.
Remaining failing test is from that bug https://github.com/ugent-library/biblio-backoffice/issues/1386, because cypress still tests the dev branch – this will be fixed later.

### CODE REVIEW

File: cypress/support/commands/extract-biblio-id.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ export default function extractBiblioId(subject: undefined | JQuery<HTMLElement>
 
     cySubject = cy.wrap(subject, { log: false })
   } else {
-    cySubject = cy.get('.list-group-item-main', { log: false })
+    cySubject = cy.get('.c-thumbnail-and-text', { log: false })
```
Comment (verheyenkoen) [inline cypress/support/commands/extract-biblio-id.ts:14]: I would replace this with this selector: `.card > .card-body > .list-group > .list-group-item`. Did some quick analysis and that should work during import and in the overview for selecting the individual publications/datasets.

**Comment by verheyenkoen** [comment]
Remaining failing test is from an unrelated bug (issue yet to be created).

**Comment by mietcls** [comment]
Should be fixed now

**Comment by verheyenkoen** [comment]
> I'll create a new PR to fix the bug, then it will pass. However, should this block the PR? This workflow might need a change.

I don't think the failing tests blocks merging the PR. The button is not disabled anyway.

**Comment by mietcls** [comment]
Okay, I will not adapt the PR in that case. We can solve both in this one.

**Review by verheyenkoen — APPROVED** [review]

**Review by nics — APPROVED** [review]

**Pull request merged**